cli/man/splinter-cert-generate.1.md

```diff
@@ -0,0 +1,95 @@
+% SPLINTER-CERT-GENERATE(1) Cargill, Incorporated | Splinter Commands
+
+NAME
+====
+
+**splinter cert generate** — Generates test certificates and keys for running
```

Comment:
Please change to splinter-cert-generate.  (This is a style change from my earlier recommendation, as noted in my comment on `cli/man/TEMPLATE.1.md.example`.) Thanks, and sorry for the extra work.

Reply:
updated